browser-interface/.gitignore

```diff
@@ -82,6 +82,7 @@ test/**/*.js
 test/**/*.js.map
 dist
 !packages/shared/world/runtime-7/sourcemap/source-map@0.7.4.js
+/protocol-temp/
```

Comment:
This folder can remain if the build process is interrupted before the end.
Its brief appearance has also been confusing my own file monitoring process.